pyproject.toml

```diff
@@ -27,8 +27,11 @@ classifiers = [
 requires-python = ">= 3.11, < 4"
 dependencies = [
   "frequenz-api-common >= 0.5.0, < 0.6.0",
```

Comment:
This needs to be removed after Tali updates the proto specs

Reply:
done.